CRM/Member/Form.php

```diff
@@ -78,6 +77,23 @@ class CRM_Member_Form extends CRM_Contribute_Form_AbstractEditPayment {
   public $_priceSetId;
```

Comment:
My apologies for missing the replies here, something happened with my e-mail filters.  @JoeMurray / @eileenmcnaughton  the history is getting a bit unwieldy on this.  I'm not sure which comment we are referring to.

To make things better (or maybe worse) I've just rebased / pushed --force onto this branch, so I'm not sure if that might have affected line numbers.  Is it worth moving into a fresh PR?